### ISSUE TITLE
Add StripeFileUpload model and tests

### ISSUE THREAD

**Issue by joshsmith** [code-corps/code-corps-api#569]
# Problem

We'll need a `StripeFileUpload` that optionally belongs to a `StripeConnectAccount` (in the db), but has a `create_identity_document_changeset/2` that does verify the association.

The model should have a:

- `id_from_stripe`
- `created` (in Stripe)
- `size`
- `purpose`
- `url`
- `type`

See: https://stripe.com/docs/file-upload

**Comment by snewcomer** [comment]
@JoshSmith does this need a controller as well?

**Comment by joshsmith** [comment]
@snewcomer yes but it can be done as a separate issue, #570. I'd like to do them as separate PRs ideally.

**Comment by joshsmith** [comment]
Partly this is because #570 is blocked by implementation inside `stripity_stripe`, which we'll also have to do.

**Comment by joshsmith** [comment]
See https://github.com/code-corps/stripity_stripe/issues/154 for reference on that.

**Comment by snewcomer** [comment]
Does this also mean it also has a `create_changeset` function or just the `create_identity_document_changeset/2`?

**Comment by joshsmith** [comment]
Closed by #574
